doc/user/rbac.md

```diff
@@ -4,9 +4,13 @@ If RBAC is in place, users need to setup RBAC rules for etcd operator. This doc
 
 ## Quick setup
 
-If you just want to play with etcd operator, there is a quick setup.
```

Comment:
Just get rid of the "admin" setting?
Now the script is there, we should  just use it for "quick setup", right?